packages/snap/README.md

```diff
@@ -1,8 +1,60 @@
-# TypeScript Example Snap
+# Gas Snap
 
-This Snap demonstrates how to develop a Snap with TypeScript.
+A MetaMask snap for sending Celo transactions with alternate gas currencies
 
-## Notes
+[![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
-- Babel is used for transpiling TypeScript to JavaScript, so when building with the CLI,
-  `transpilationMode` must be set to `localOnly` (default) or `localAndDeps`.
+## Install
+
+```
+yarn add @celo/gas-snap
+```
+
+## Usage
+
+To interact with this Snap, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features.
+
+Then use Snap ID to connect to this Snap:
+
+```javascript
+const snapId = 'npm:@celo/gas-snap';
```

Comment:
this is a place holder, as snap has not yet been publlished. waiting for npm access to fix that.

Reply:
best would be to talk with @dckesler  or @pputman-clabs about getting automated publishing set up

Reply:
yea thats a good idea, for now we want to deploy manually via cli